console/client/src/features/modules/ModulesSidebar.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react'
+import { Listbox } from '@headlessui/react'
+import { EyeIcon } from '@heroicons/react/20/solid'
+import { Module } from '../../protos/xyz/block/ftl/v1/console/console_pb'
+import { VerbId, ZoomCallbacks } from './modules.constants'
+import { getNames } from './modules.utils'
+import { classNames } from '../../utils'
+import { backgrounds, borders, colors } from './components'
+
+interface MapValue {
+  deploymentName: string
+  id: string
+  verbs: Set<VerbId>
+  queriedVerbs: Set<VerbId>
+}
```

Comment:
`MapValue` seems a bit generic, is there a better name we could use here? Maybe like `DeploymentVerbs` or something? I'm cool if this makes sense as well.

Reply:
Honestly I started generic and never swapped in something better to describe it. Let's rename.